modules/hcp-import-limitations.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-import.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="hcp-import-limitations_{context}"]
+= Limitations of managing imported hosted clusters
+
+Hosted clusters are automatically imported into the local {mce}, unlike a standalone {product-title} or third party clusters. Hosted clusters run some of their agents in the _hosted mode_ so that the agents do not use the resources of your cluster. If you choose to automatically import hosted clusters, you can update node pools and the control plane in hosted clusters by using the `HostedCluster` resource on the management cluster. To update node pools and a control plane, see "Updating node pools in a hosted cluster" and "Updating a control plane in a hosted cluster".
+
+You can import hosted clusters into a location other than the local {mce-short} by using the {mce-short} or {rh-rhacm-first}. You must use the following options to update hosted clusters:
+
+* The {mce} console or the command-line interface of the management cluster, if you use the {mce} to import a hosted cluster. To update a hosted cluster by using the {mce} console, see "Updating a hosted cluster by using the multicluster engine Operator console".
+* A policy, if you use the {rh-rhacm} hub cluster to import a hosted cluster. For more information, see "Configuring settings for automatic import".
+
+The {mce} console can directly access the `HostedCluster` and `NodePool` resources. {rh-rhacm} agent runs inside the managed cluster. In {hcp}, the control plane handles the updates, which is present outside the hosted clusters. Unlike hosted clusters, you can update standalone {product-title} clusters by using the `ClusterVersion` resource, which is present inside an {product-title} cluster. To update an {product-title} cluster, see "Updating a cluster by using the CLI".
```

Comment:
I think this paragraph is confusing and already mentioned in previous lines. We should remove this.

Reply:
Addressed